doctr/datasets/vocabs.py

```diff
@@ -13,7 +13,7 @@
     "digits": string.digits,
     "ascii_letters": string.ascii_letters,
     "punctuation": string.punctuation,
-    "currency": "£€¥¢฿",
+    "currency": "£€¥¢฿₴",
```

Comment:
Unfortunately this would break our current models so we need to add it directly to the ukrainian vocab